app/components/curriculum-inventory/sequence-block-overview.hbs

```diff
@@ -299,6 +299,58 @@
               {{/if}}
             </span>
           </div>
+          <div class="block starting-academic-level" data-test-starting-academic-level>
+            <label for="academic-level-{{templateId}}">{{t "general.startLevel"}}:</label>
+            {{#if @canUpdate}}
+              <EditableField
+                @value={{this.startingAcademicLevel.name}}
+                @save={{perform this.changeStartingAcademicLevel}}
+                @close={{this.revertStartingAcademicLevelChanges}}
+              >
+                <select
+                  id="starting-academic-level-{{templateId}}"
+                  {{on "change" this.setStartingAcademicLevel}}
+                >
+                  {{#each (sort-by "level" this.academicLevels) as |obj|}}
+                    <option value={{obj.id}} selected={{eq obj.id this.startingAcademicLevel.id}}>{{obj.name}}</option>
+                  {{/each}}
+                </select>
+                {{#each (await (compute this.getErrorsFor "startingAcademicLevel")) as |message|}}
+                  <span class="validation-error-message">
+                    {{message}}
+                  </span>
+                {{/each}}
+              </EditableField>
+            {{else}}
+              <span>{{this.startingAcademicLevel.name}}</span>
+            {{/if}}
+          </div>
+          <div class="block ending-academic-level" data-test-ending-academic-level>
+            <label for="academic-level-{{templateId}}">{{t "general.endLevel"}}:</label>
+            {{#if @canUpdate}}
+              <EditableField
+                @value={{this.endingAcademicLevel.name}}
+                @save={{perform this.changeEndingAcademicLevel}}
+                @close={{this.revertEndingAcademicLevelChanges}}
+              >
+                <select
+                  id="ending-academic-level-{{templateId}}"
+                  {{on "change" this.setEndingAcademicLevel}}
+                >
+                  {{#each (sort-by "level" this.academicLevels) as |obj|}}
+                    <option value={{obj.id}} selected={{eq obj.id this.endingAcademicLevel.id}}>{{obj.name}}</option>
+                  {{/each}}
+                </select>
+                {{#each (await (compute this.getErrorsFor "endingAcademicLevel")) as |message|}}
```

Comment:
Can be replaced with `<ValidationError @errors={{get-errors-for this.endingAcademicLevel}} />``

Reply:
:heavy_check_mark: